### PR TITLE
Super Team

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by mujz — APPROVED** [review]
Nice one Super team! It looks good to me. 
I'm guessing that you've split the work for the part you're doing until Tuesday/Wednesday and you're going to split among yourselves the remainder of the website later? Because there's still the "Enjoy The Best Cuisine" section, the "Customer" section, "Select a Date & Book Now" section, "How to Find & Contact Us" section, and footer section. But I think it would make sense to leave these out as the second half of the project.

**Comment by ghost** [comment]
Thanks
Yeah we split oure project in two part we finish first part before 5 may
and finish second part pefore 8 may

**Comment by ghost** [comment]
we are finish

File: Super Team/main.css [hunk 1]
```diff
@@ -0,0 +1,605 @@
...
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+.card{
+  padding: 0px 50px !important;
+}
+.style-h {
+  font-family: 'Cormorant Garamond', serif;
+}
+.padd{
```
Comment (osamaakb) [inline Super Team/main.css:194]: try to name the classes in a way that the class name speaks for what it do

File: Super Team/main.css [hunk 2]
```diff
@@ -0,0 +1,605 @@
...
+  padding: 0;
+}
+.seccolumn{
+  padding: 200px 45px 200px 140px !important;
+}
+.myrow{
+  margin: 0;
+  margin-top: 30px;
+}
+.seccolumn{
```
Comment (osamaakb) [inline Super Team/main.css:247]: this class is a duplication 🙈

File: Super Team/main.css [hunk 3]
```diff
@@ -0,0 +1,605 @@
...
+}
+.lightgray-text{
+  color: rgb(166, 166, 166);
+}
+.style-h{
+  padding-top: 4%;
+  color:rgb(62, 62, 62);
+}
+.card-size{
+  margin: 3% auto !important;
```
Comment (osamaakb) [inline Super Team/main.css:289]: try to avoid using % with margin and use pixels

File: Super Team/main.css [hunk 4]
```diff
@@ -0,0 +1,605 @@
...
+  width: 83% !important;
+  padding: 0 !important;
+}
+.card-border{
+    border-style: none !important;
```
Comment (osamaakb) [inline Super Team/main.css:294]: you are using important a lot, do you really need it this much? I don't think so

File: Super Team/main.html [hunk 5]
```diff
@@ -0,0 +1,632 @@
...
+        </div>
+      </div>
+      <h5>Q U A L I T Y</h5>
+      <h1 class="text-left">Resturant</h1>
+      <h1 class="text-right">&Dishes</h1>
+    </div>
+  </div>
+</section>
+<div class="pink"></div>
+<script>
```
Comment (osamaakb) [inline Super Team/main.html:131]: its not a good practice to use the script tag in between HTML lines

File: Super Team/main.html [hunk 6]
```diff
@@ -0,0 +1,632 @@
...
+      </div>
+      <div class="col-12 d-flex justify-content-center rest-addres" style="margin: 0;">
+        <div class="col-md-4 d-flex justify-content-center " style="margin: 0;">
+          <p style="margin: 0;">Erbil - Resturant</p>
+        </div>
+      </div>
+      <br>
+      <div class="col-12 d-flex justify-content-center rest-addres" style="margin: 0;">
+        <div class="col-md-4 d-flex justify-content-center" style="margin: 0;">
+          <p style="margin: 0;"> <strong>Business Phone:</strong></p >
```
Comment (osamaakb) [inline Super Team/main.html:523]: avoid using inline styles like here

**Review by osamaakb — COMMENTED** [review]
Ammmmaaaazzzzzing work super team 🎉.
I really liked the results.
your CSS is really long and still, you are using bootstrap too, so I would suggest you guys get aligned to either use CSS or bootstrap.
there was a lot of JavaScript which I'm not sure if you are understanding it well or just implementing it there 😬.
also, I left you some comments hopefully you get use of them in your next project.

File: Super Team/main.css (hunk 4 above)
Comment (ghost) [reply Super Team/main.css:294]: without using it ,its doesn't work

**Comment by ghost** [comment]
thanks
for that, we want using bootstrap but we do not have a good knowledge for using bootstrap well

**Pull request closed without merging**